### DISCUSSION TITLE
Why does AlDente track our internet browsing activity?

### DISCUSSION THREAD

**Discussion by Tanz0rz** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1254] [General]
This is very clearly legal speak for using this app as spyware by the developers. At the very least, this should be completely absent from the paid versions

https://apphousekitchen.com/policy/

![tracking](https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/assets/6529295/effae8e8-1f77-455d-a2ff-c4f5b30ee90a)

**Comment by Tanz0rz** [comment]
bump

**Reply by Tanz0rz** [reply]
bump

**Comment by Tanz0rz** [comment]
bump

**Comment by masterzone3d** [comment]
If you don't want being tracked, stop surfing the web

**Reply by Tanz0rz** [reply]
?

**Reply by masterzone3d** [reply]
Why are you interested to the website cookies?

**Reply by Tanz0rz** [reply]
Are you tracking our browsing history or browsing activity on websites other than your website?

**Reply by Tanz0rz** [reply]
bump

**Comment by MatthiasKerbl** [comment]
Hi @Tanz0rz,

Sorry for the late reply. We are not tracking any user data with AlDente from our side. However, depending on which version of AlDente you are using and if you bought it through Paddle, or you are using it through Setapp, some usage data might be shared. If you bought AlDente through Paddle, the license status will be checked on a regular base to avoid illegal usage of AlDente. Furthermore, if you are using AlDente Pro through Setapp, some usage data will be collected so that Setapp can determine which apps are being used to pay developers fairly. You can read more about [Paddles](https://www.paddle.com/) and [Setapps](https://setapp.com/) privacy policy on their websites.

Hope that clarifies things better.

Best,
Matthias

**Reply by Tanz0rz** [reply]
Ok, so to clarify, if we are using the desktop app downloaded from your website directly, on both the free and paid versions, the Al Dente app does not do any tracking of our data or website browsing activity?

**Reply by Tanz0rz** [reply]
bump

**Reply by Tanz0rz** [reply]
bump

**Reply by Tanz0rz** [reply]
bump

**Reply by MatthiasKerbl** [reply]
> Ok, so to clarify, if we are using the desktop app downloaded from your website directly, on both the free and paid versions, the Al Dente app does not do any tracking of our data or website browsing activity?

The only part of the AlDente App, if you download it from our website, that tracks user data is the license verification part from paddle.com. As described earlier, this is necessary to avoid illegal usage of AlDente. You can read more about [Paddles](https://www.paddle.com/) privacy policy on their website. Furthermore, AlDente does not do any tracking of browsing activity. However, if you go to our website through AlDente, some browsing activity is tracked. You can read more about it in our [privacy policy](https://apphousekitchen.com/policy/). Nonetheless, there is no cross-tracking between AlDente and our website.

Best,
Matthias

**Reply by Tanz0rz** [reply]
Thanks!
